tooling/README.md

```diff
@@ -0,0 +1,63 @@
+Table of Contents
+=================
+
+* [Introduction](#introduction)
+* [ssh-vm](#ssh-vm)
+   * [Usage](#usage)
+   * [The passwordstore database](#the-passwordstore-database)
+
+
+# Introduction
+
+Project tools.
```

Comment:
- Can you elaborate a little bit more here around `Project tools`.
- Dont forget to also include a prereq section as it is needed that password store is installed, keys signed, ...

Reply:
I added some infor on the passwordstore product and the team  project database but didn't want to duplicate the information already existing on that project.

Regarding the tooling project, as it already existed, I moved the new script to a new `tools` project.